inheritance/src/main/java/inheritance/Library.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package inheritance;
+
+public class Library {
```

Comment:
I don't recommend nesting classes like this unless the parent class only ever uses the sub class internally.